net/mbusd/Makefile

```diff
@@ -0,0 +1,53 @@
+#
+# Copyright (C) 2014 OpenWrt.org
```

Comment:
This should either be you, or omitted, unless you have copyright assignment agreements with openwrt, and unless they were done in 2014.